### ISSUE TITLE
Option to suppress languages in --help

### ISSUE THREAD

**Issue by Awebb** [fosskers/aura#927]
**Issue:** When using --help, there is a whole screen with --_language_ options and the user has to scroll up that whole screen to see something useful.

**Request:** Add an option to the config.toml that suppresses the output of those --_language_ options for better readability.
```
[general]
helplangs = false
```

**Optional idea:**
Put all languages behind a --language _language_ switch.

**Comment by fosskers** [comment]
Try `aura -h` instead, you'll notice that the language flags don't appear. This is due to:
```
hide_short_help = true
```
being set on those flags. Note that `hide_long_help` also exists, but I chose not to activate that so that the flags would appear at least somewhere. It may be that most people change Aura's language through `LANG` or Aura's config directly, so the flags may not be as useful.

**Comment by Awebb** [comment]
That does it for me personally! I did not expect the short and long version of a command to behave differently, though. Is this now common practice and I didn't notice or should we perhaps add this to the man page?

**Comment by fosskers** [comment]
I wouldn't say it's common practice, but I do want the language flags to be hidden from whatever users would tend to use most. You seemed to have reached naturally for `--help` instead of `-h`. What do you think other users' tendency might be?

**Comment by Awebb** [comment]
I've reached for --help, because I've been doing a lot of scripting lately (not necessarily with aura) and I try to "let the code document itself", so I only use short commands whenever I'm really doing something multiple times a day/a week.

I don't have a real objection against the short option not just being an alias to the long option. I simply did not expect it. I've just noticed the "see more with --help" hint under --help and -h, so it's not completely undocumented. How about we add the same note to the man page?
